### PR TITLE
Form validation on Fill a Loan page

### PR DESCRIPTION
**Pull request by jmul-dev**
This PR introduces the following changes:

- Validate whether or not user entered JSON string is valid
- In addition, it will also check whether the entered JSON string has all of the required debt order fields from request order

### CODE REVIEW

**Comment by jmul-dev** [comment]
screen shot 2018-04-04 at 3 06 29 pm

**Comment by jmul-dev** [comment]
screen shot 2018-04-04 at 3 05 24 pm

File: src/modules/FillLoan/FillLoanEmpty/index.tsx [hunk 1]
```diff
@@ -33,26 +34,102 @@ class FillLoanEmpty extends React.Component<{}, State> {
         browserHistory.push(`/fill/loan?${encodeUrlParams(loanRequest)}`);
     }
 
+	validateForm(formData: any, errors: any) {
+		try {
+			const loanRequest = debtOrderFromJSON(formData.loan.loanRequest);
```
Comment (nadavhollander) [inline src/modules/FillLoan/FillLoanEmpty/index.tsx]: I think you ought to use something like JSON schema here instead -- this is very messy / hard to read.  Please clean up or find a more compact way of doing this.

**Comment by jmul-dev** [comment]
This is fixed

**Pull request merged**